.eslintrc.js

```diff
@@ -2,11 +2,9 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    jest: true,
```

Comment:
Out of curiosity, what command did you run that replaced all the double quotes here with single quotes?